index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet"/>
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="hamburge-menu">
+        <a href="#">John Doe</a>
+        <img src="./img/Menu.png" alt="Hamburger-Menu" />
+      </nav>
+    </header>
+    <main>
+      <div class="message">
+        <h1>Hey There. <br />I’m Alex</h1>
+        <span class="title">I’am a Software Developer</span>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
+        <ul>
+          <li>
+            <a href="http://github.com/mrbhi"
+              ><img class="social-icon-btn" src="./img/GitHub.png" alt="GitHub"
+            /></a>
+          </li>
+          <li>
+            <a href="http://linkedin.com"
+              ><img
+                class="social-icon-btn"
+                src="./img/Linkedin.png"
+                alt="Linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="http://angelist.com"
+              ><img
+                class="social-icon-btn"
+                src="./img/Angelist.png"
+                alt="Angelist"
+            /></a>
+          </li>
+          <li>
+            <a href="http://twitter.com/ADEX8600"
+              ><img
+                class="social-icon-btn"
+                src="./img/Twitter.png"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="http://medium.com"
+              ><img class="social-icon-btn" src="./img/Medium.png" alt="Medium"
+            /></a>
+          </li>
+        </ul>
+      </div>
+      <div class="main-image">
+        <img class="bg" src="./img/Bg.png" alt="Background" />
+      </div>
```

Comment:
- [ ] Please make sure that for your **headline section** you add the proper background image as background images in your CSS and do not use it as img tags because if an image is not part of the content (aesthetic reasons) use CSS background-image. :smile: